data/repository-impl/src/main/java/io/github/droidkaigi/confsched2019/data/repository/DataWifiConfigurationRepository.kt

```diff
@@ -0,0 +1,19 @@
+package io.github.droidkaigi.confsched2019.data.repository
+
+import io.github.droidkaigi.confsched2019.data.device.WifiManager
+import io.github.droidkaigi.confsched2019.model.WifiConfiguration
+import kotlinx.coroutines.coroutineScope
+import javax.inject.Inject
+
+class DataWifiConfigurationRepository @Inject constructor(
```

Comment:
I think that you do not have to use the Repository except when you want to cache it. 👀